src/guide/components.md

```diff
@@ -4,6 +4,11 @@ type: guide
 order: 12
 ---
 
+##What are Components?
+Components are one of the most powerful features of Vue.js, helping you extend basic HTML elements and create reusable code. At a high level, Components are custom elements that Vue.js’ compiler attaches a specified behavior, or even transforms the elements and their children.
```

Comment:
minor grammar issue: probably should be "Components are custom elements that Vue.js’ compiler would attach specified behavior to"